content/WebAuthn/WebAuthn_Walk-Through.adoc

```diff
@@ -0,0 +1,283 @@
+== WebAuthn Walk-Through
+Build an app that enables your users to register and authenticate through WebAuthn. To implement the WebAuthn security protocol in your app, use the link:https://www.w3.org/TR/webauthn/[W3C Web Authentication API] specification and Yubico’s link:https://github.com/Yubico/java-webauthn-server/tree/master/webauthn-server-demo[Java WebAuthn Server].
```

Comment:
consider leaving off 'security protocol' - > To implement WebAuthn in your app

Reply:
Removed some time ago